.github/workflows/actions.yml

```diff
@@ -0,0 +1,46 @@
+name: Workflow
+on:
```

Comment:
This configuration seems to run `Workflow` with every new issue **and** pull request. _This is not in the **spirit** of issue #738, which aims at both functionality and efficiency._

Is there a way to have separate sections in the same file, one that works for PRs only and another for issues only?

If not, then create two files.

1. `pull-request.yml` - For PRs only. Include:
    1. the `PR Greeting` [with an appropriate message](https://github.com/actions/first-interaction), 
    2. `Linter`
    3. `Count-lines-of-code`. 
1. `issues.yml` - For issues only. Include adding:
      1. a new `issue specific greeting` 
      2. the `unapproved` label.

Give a meaningful name to each block in the job so that you can have a more information in the form of a sentence to give more information to the user. At the moment they can only see the names of the YAML keys namely: `Opened-issues`, `Greeting`, `Linter` and  `Count-lines-of-code`. See image below:

![image](https://user-images.githubusercontent.com/16875803/116325326-bfc0df80-a776-11eb-9d3d-d5cd37182545.png)

Here is the example in the [GitHub Actions page's job section](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs) on providing names.

```yaml
jobs:
  my_first_job:
    name: My first job
  my_second_job:
    name: My second job
```